MarketExecute.php

```diff
@@ -178,6 +178,32 @@ function DefaultErrorHandler($errno, $errstr) {}
                     "ErrorMessage" => "GetAllProducts: unsupported operation"));
             break;
 
+        case "GetNextProducts":
```

Comment:
Can we call this "GetAllProducts" since it is returning products regardless of type? (There is already a "GetAllProducts", can just paste the code there).

In addition to this function, can we also give "GetProductsByType" and "GetProductsByUserId" the pagination functionality? We should make this a standard thing.